packages/blade/src/components/Drawer/types.ts

```diff
@@ -38,6 +38,14 @@ type DrawerProps = {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   initialFocusRef?: React.MutableRefObject<any>;
+
+  /**
+   * If `true`, the DrawerBody will be rendered only when it becomes active.
+   * Set to `false` to keep DrawerBody in DOM
+   *
+   * @default true
+   */
+  isLazy?: boolean;
```

Comment:
In case they are doing `document.querySelector` somewhere, or if they want to render a portal container inside Drawer (portal container has to exist in DOM all time) which was the usecase for us.

We have similar prop in Tabs so exposed it to consumers